src/export/bops/model.ts

```diff
@@ -0,0 +1,146 @@
+import { Edges, NodeId, Node, Crumb } from "../../types";
+
+// Less strict FlowGraph and Breadcrumbs types which replace Value with any allows us to port over BOPS logic as-is, with goal to tighten over time
+interface LooseNode extends Node {
+  data?: Record<NodeId, any>;
+}
+
+export type LooseFlowGraph = {
+  _root: {
+    edges: Edges;
+  };
+  [key: string]: LooseNode;
+};
+
+interface LooseCrumb extends Crumb {
+  data?: Record<string, any>;
+}
+
+export type LooseBreadcrumbs = {
+  [key: string]: LooseCrumb;
+}
```

Comment:
I've defined these `Loose...` types to get around some issues using the existing `Value` type - they let us get to payload-parity much quicker here, and my intention is that we'll remove them over time as we revisit/refactor/better generalise key methods like `formatProposalDetails` to use the new stricter types.